### PR TITLE
Change the location of Iceberg

### LINKED ISSUE #30: Change the location of Iceberg
Now, Iceberg use the same location as Hive. This may be misleading for newcomers. We change it to a new path

### PR DESCRIPTION
**Pull request by qqqttt123**
We add a new path: /user/iceberg/warehouse. 
Modify the configurations of each components.
Fix #30

### CODE REVIEW

**Comment by jerryshao** [comment]
Did you test it locally?

**Comment by qqqttt123** [comment]
> Did you test it locally?

Yes, I have tested locally.  I ran the SparkSQL and checked the web page.

File: docker-compose.yaml [hunk 1]
```diff
@@ -95,7 +98,7 @@ services:
     image: mysql:8.0
     container_name: playground-mysql
     ports:
-      - "3306:3307"
+      - "3306:3306"
```
Comment (qqqttt123) [inline docker-compose.yaml:101]: Fix the port mapping by the way.

**Comment by qqqttt123** [comment]
Remind @jerryshao again.

**Pull request merged**